### DISCUSSION TITLE
[Feature request] Format date as result of datetime operation

### DISCUSSION THREAD

**Discussion by maveec** [grafana/grafana#40929] [General]
Is it possible to support datetime operations - and subsequent formatting - natively in Grafana? 
I would like to be able to format a date resulting from a datetime operation in which an interval is added to the date variable.

The syntax would be something along the lines:

${(__to+1d): date:YYYY-MM-DD}:
In this case I want to add 1 day to __to and then format it. 

While this functionality is less useful when data are retrieved via a DB, as the dialects support these operations, this functionality becomes useful for plugins that do not support datetime operations or datetime formatting. e.g. json API.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
